src/main/java/com/nylas/EventQuery.java

```diff
@@ -111,5 +132,49 @@ public EventQuery endsAfter(Instant endsAfter) {
 		this.endsAfter = endsAfter;
 		return this;
 	}
-	
+
+	/**
+	 * Add required metadata key to the query.
+	 * Returned events must include a metadata property with this key.
+	 * This method can accept multiple strings or be invoked multiple times
+	 * for the event to have multiple metadata keys. Each time this is called it
+	 * adds another required value rather than replacing previous required keys.
```

Comment:
Rajeev supplied answers that multiple `metadata` filters (of the same type or different types) are ORed together, unlike other filters.  We need to update the javadoc on the three methods here to reflect that.  Here's a suggestion:

```
/**
 * Return events with metadata containing a property having the given key.
 * 
 * If multiple instances of metadata methods are invoked
 * (any combination of calls to metadataKey, metadataValue, or metadataPair),
 * then this query will return events which match ANY one of them.  
 */
```

Reply:
Awesome, added in now.